src/app/modules/main-module/components/main/main.component.scss

```diff
@@ -31,7 +26,22 @@
   display: flex;
   align-items: center;
 
-  &.selected{
-    border: 3px solid yellow;
+  &.selected {
```

Comment:
aquí si tenéis que usar los colores en teclado y casilla, las &.XXXX sobran por no repetir código, usad las de fuera